test/test_ignore_folders.py

```diff
@@ -0,0 +1,49 @@
+import json
+from pathlib import Path
+from typing import List
+
+import pytest
+from _pytest.tmpdir import TempPathFactory
+
+from exasol_script_languages_container_ci.lib.ci import check_if_need_to_build
+from test.fixtures import tmp_test_dir
+import git
+
+
+def commit_files(repo: git.Repo, repo_path: Path, files_to_commit: List[str]):
+    for file in files_to_commit:
+        (repo_path / file).parent.mkdir(parents=True, exist_ok=True)
+        open(repo_path / file, 'w').close()
```

Comment:
what is this line for? Why open and do nothing?

Reply:
I added a comment. I hope it's now clearer what's going on...